packages/medusa-js/src/key-manager.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * `KeyManager` holds API keys in state.
+ */
+class KeyManager {
+  private publishableApiKey: string | null = null
+
+  /**
+   * Set a publishable api key to be sent with each request.
+   */
+  public registerPublishableApiKey(key: string) {
+    this.publishableApiKey = key
+  }
+
+  /**
+   * Retrieve the publishable api key.
+   */
+  public getPublishableApiKey() {
```

Comment:
suggestion: would a getter be enough instead?
```
get publishableApiKey(): string {
  return this. publishableApiKey_ ?? ""
}
```

Reply:
Out of curiosity, are there any significant differences between the two, or is it just a matter of choosing one and sticking with it?

I am asking because it might make sense to add to our conventions doc, if that is the case.

Reply:
there is not much difference except that with a getter you don't have to invoke it. also, it seams more natural when we have private/protected member to expose getter/?setter for that property instead of a method to invoke to get that prop. It make it more natural for the consumer.